Description: Upstream changes introduced in version 1.83-1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 lpc21isp (1.83-1) unstable; urgency=low
 .
   * Initial release
 .
 The person named in the Author field signed this changelog entry.
Author: Felix Ruess <felix.ruess@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- lpc21isp-1.83.orig/Makefile
+++ lpc21isp-1.83/Makefile
@@ -1,3 +1,5 @@
+prefix=/usr/local
+
 all:      lpc21isp
 
 GLOBAL_DEP  = adprog.h lpc21isp.h lpcprog.h lpcterm.h
@@ -31,3 +33,6 @@ lpc21isp: lpc21isp.c adprog.o lpcprog.o
 
 clean:
 	$(RM) adprog.o lpcprog.o lpcterm.o lpc21isp
+
+install:
+	cp lpc21isp $(DESTDIR)$(prefix)/bin
